graphein/protein/config.py

```diff
@@ -6,16 +6,41 @@
 # Code Repository: https://github.com/a-r-j/graphein
 from __future__ import annotations
 
+from functools import partial
 from pathlib import Path
-from typing import Callable, List, Optional, Union
+from typing import Any, Callable, List, Optional, Union
 
+from deepdiff import DeepDiff
+from deepdiff.operator import BaseOperator
 from pydantic import BaseModel
 from typing_extensions import Literal
 
 from graphein.protein.edges.distance import add_peptide_bonds
 from graphein.protein.features.nodes.amino_acid import meiler_embedding
 
 
+def partial_functions_equal(func1, func2):
```

Comment:
Some docstrings would be super helpful here

Reply:
Done